lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -0,0 +1,77 @@
+require 'libxml'
+
+module Twilio
+  module TwiML
+    class TwiMLError < StandardError
+    end
+
+    class TwiML
+      # Generate getter/setter methods
+      attr_accessor :name
+      attr_accessor :indent
+
+      def initialize(indent: false, **keyword_args)
+        @name = self.class.name.split('::').last
+        @indent = indent || false
```

Comment:
Dont think this is necessary given that you set indent's default value to false above.